tools/autograd/derivatives.yaml

```diff
@@ -303,22 +335,11 @@
 - name: cholesky_inverse(Tensor self, bool upper=False) -> Tensor
   self: cholesky_inverse_backward(grad, self, upper, result)
 
-- name: fbgemm_linear_int8_weight_fp32_activation(Tensor input, Tensor weight, Tensor packed, Tensor col_offsets, Scalar weight_scale, Scalar weight_zero_point, Tensor bias) -> Tensor
```

Comment:
These are plain wrong because:
- self is not an input for these functions so these formulas make no sense
- these functions don't have a dispatch in derivative.yaml so this cannot be called (codegen does not fail on master because the formulas point to a non-argument and so the formulas as ignored)